src/yetibot/core/commands/alias.clj

```diff
@@ -22,7 +22,8 @@
           ;; evaluation of piped expressions (i.e. %s instead of $s)
           expr (binding [*subst-prefix* method-like-replacement-prefix]
                  (str command/config-prefix (pseudo-format-n cmd args)))
-          results (record-and-run-raw expr user yetibot-user
+          resolved-users nil ;; TODO
```

Comment:
I'm not sure aliases would ever mention users. Either way, we can defer this to a later PR if we want.